skills/ralph/SKILL.md

```
@@ -91,29 +91,34 @@ Stories execute in priority order. Earlier stories must not depend on later ones
 - "Tests pass"
 
 ### Bad criteria (vague):
+
 - "Works correctly"
 - "User can do X easily"
 - "Good UX"
 - "Handles edge cases"
 
 ### Always include as final criterion:
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)